lib/taskana-core/src/test/java/acceptance/taskrouting/TaskRoutingAccTest.java

```diff
@@ -3,92 +3,101 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
 
-import acceptance.AbstractAccTest;
+import acceptance.DefaultTestEntities;
+import acceptance.taskrouting.TaskRoutingAccTest.TaskRoutingProviderForDomainA;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
+import testapi.TaskanaInject;
+import testapi.TaskanaIntegrationTest;
+import testapi.WithServiceProvider;
 
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.api.TaskanaEngine;
 import pro.taskana.common.api.exceptions.InvalidArgumentException;
-import pro.taskana.common.test.security.JaasExtension;
 import pro.taskana.common.test.security.WithAccessId;
-import pro.taskana.task.api.TaskCustomField;
+import pro.taskana.spi.routing.api.TaskRoutingProvider;
 import pro.taskana.task.api.TaskService;
 import pro.taskana.task.api.models.Task;
-import pro.taskana.task.internal.models.TaskImpl;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
 
-/** Acceptance test for all "create task" scenarios. */
-@ExtendWith(JaasExtension.class)
-class TaskRoutingAccTest extends AbstractAccTest {
+@WithServiceProvider(
+    serviceProviderInterface = TaskRoutingProvider.class,
+    serviceProviders = TaskRoutingProviderForDomainA.class)
+@TaskanaIntegrationTest
+class TaskRoutingAccTest {
 
-  @WithAccessId(user = "admin")
-  @Test
-  void testCreateTaskWithoutWorkbasketAndVoidNewTaskMethod() throws Exception {
```

Comment:
where did this go / is this nonsense?

Reply:
Most of this class was nonsense.
The Test is here: should_SetWorkbasketForTask_When_TaskRouterDeterminesWorkbasket